modules/@apostrophecms/notification/index.js

```diff
@@ -238,9 +236,10 @@ module.exports = {
               throw self.apos.error('invalid');
             }
             start = Date.now();
+            modifiedOnOrSince = req.query.modifiedOnOrSince && self.apos.launder.date(req.query.modifiedOnOrSince);
```

Comment:
Hmm, that would be literally a date, it wouldn't allow for date and time. So it would send a lot more redundant notifications than we intended. I think you should launder it by running it through the date constructor and treating it as null if it throws an exception.

Reply:
After the laundering step, here is a `modifierOnOrSince` example: `2020-08-26T17:18:22.453Z`. Is it not good enough?

Reply:
I think you may have found a bug in launder.date 😆 But this is really not what it is supposed to do. So please do not rely on it. Do this:

```
if (req.query.modifiedOnOrSince) {
  try {
    modifiedOnOrSince = new Date(req.query.modifiedOnOrSince);
  } catch (e) {
  }
```